### DISCUSSION TITLE
Implement PyDocLint

### DISCUSSION THREAD

**Discussion by Goldziher** [astral-sh/ruff#12324] [Ideas]
[pydoclint](https://github.com/jsh9/pydoclint) is a nifty tool for linting doc strings. It would be great to have this functionality in ruff.

**Comment by dhruvmanila** [comment]
Hey, thanks for the feature request. Docstring linting is tracked here: https://github.com/astral-sh/ruff/issues/458 and there's also a mention of `pydoclint` in this comment: https://github.com/astral-sh/ruff/issues/458#issuecomment-2122773827

**Reply by Goldziher** [reply]
cool, ill add my 5 cents
